Specs/DateTime/Dutch/DateExtractor.json

```diff
@@ -2585,5 +2585,77 @@
     "Comment": "1/4 shouldn't recognized as date here",
     "NotSupportedByDesign": "javascript,python,java",
     "Results": []
+  },
+  {
+    "Input": "Ik ga terug op Sep-23-2020.",
+    "NotSupportedByDesign": "javascript,python,java",
```

Comment:
Why disable tests in the other platforms? If the changes are YAML regex only, they should work, no?

Reply:
re-enabled but only in English, French and Spanish (the other languages are implemented only in .NET)